kubernetes/overlays/dev/prompt-proto-service/prompt-proto-service.yaml

```diff
@@ -11,53 +11,70 @@ spec:
         autoscaling.knative.dev/target: "1"
         autoscaling.knative.dev/min-scale: "1"
         autoscaling.knative.dev/max-scale: "100"
+        # Update this field if `make apply` by itself doesn't make a new revision.
+        revision: "31"
+      creationTimestamp: null
     spec:
+      containerConcurrency: 0
+      containers:
+      - image: us-central1-docker.pkg.dev/prompt-proto/prompt/prompt-proto-service:latest
+        imagePullPolicy: Always
+        name: user-container
+        env:
+        - name: RUBIN_INSTRUMENT
+          value: HSC
+        - name: PUBSUB_VERIFICATION_TOKEN
+          value: rubin-prompt-proto-main
+        - name: IMAGE_BUCKET
+          value: rubin-pp
+        - name: CALIB_REPO
+          value: s3://rubin-pp-users/central_repo/
+        - name: IP_APDB
+          # Need explicit port for make_pgpass.py
+          value: usdf-prompt-processing-dev.slac.stanford.edu:5432
+        - name: DB_APDB
+          value: lsst-devl
+        - name: USER_APDB
+          value: rubin
+        - name: NAMESPACE_APDB
+          value: pp_apdb
+        - name: IP_REGISTRY
+          # Need explicit port for make_pgpass.py
+          value: usdf-prompt-processing-dev.slac.stanford.edu:5432
+        - name: DB_REGISTRY
+          value: ppcentralbutler
+        - name: USER_REGISTRY
+          value: pp
+        - name: KAFKA_CLUSTER
+          value: prompt-processing-kafka-bootstrap.kafka:9092
+        - name: PSQL_REGISTRY_PASS
+          valueFrom:
+            secretKeyRef:
+              name: usdf-pp-creds
+              key: password
+        - name: PSQL_APDB_PASS
+          valueFrom:
+            secretKeyRef:
+              name: usdf-prompt-processing-creds
```

Comment:
This looks a little weird: two secrets with very similar names for different databases.

Reply:
Yes agree.  One database uses the rubin user and the other pp.  I can update to be more specific.